### PR TITLE
Others

### PR DESCRIPTION
**Pull request by Furtif**

### CODE REVIEW

File: library/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/out/
```
Comment (Gegy) [inline library/.gitignore]: What's the reason for this addition?

File: library/build.gradle [hunk 2]
```diff
@@ -3,8 +3,8 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.7'
-        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.1"
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.3'
```
Comment (Gegy) [inline library/build.gradle]: Are the updated libraries here required?

**Review by Gegy — COMMENTED** [review]
The changes you've made look good. I've heard there are changes to the requests made by the client, such as `download_settings` being sent in every request. Could you address these?

File: library/.gitignore (hunk 1 above)
Comment (Furtif) [reply library/.gitignore]: fixed https://github.com/Grover-c13/PokeGOAPI-Java/pull/945/commits/fbd834918a46c5e6bf45ff6285f0e62fea21b2b3

File: library/build.gradle (hunk 2 above)
Comment (Furtif) [reply library/build.gradle]: fixed https://github.com/Grover-c13/PokeGOAPI-Java/pull/945/commits/c2e6fcd3d851ed025c98a184060469c675a1dd7d

File: library/.gitignore (hunk 1 above)
Comment (Gegy) [reply library/.gitignore]: You seem to have kept this, and instead removed it in the other .gitignore. Was this intended?

File: library/build.gradle (hunk 2 above)
Comment (Gegy) [reply library/build.gradle]: It wasn't a request to downgrade -- it was a question of why an update is used here, and is it in scope of the PR? Also, you seem to have a trailing tab a few lines up.

File: library/.gitignore (hunk 1 above)
Comment (Furtif) [reply library/.gitignore]: I had added this because I was manually compiling the protos nothing much

**Review by Gegy — CHANGES_REQUESTED** [review]
Before merging, this PR should address changes to API calls, which I mentioned in my first review.

**Comment by Furtif** [comment]
fix https://github.com/Grover-c13/PokeGOAPI-Java/pull/945#pullrequestreview-82560450 no dowload settings this was before i had to check challange but the check challange came back this ok this Reference on Discord channel

**Comment by Gegy** [comment]
Previous requested changes still apply.

**Comment by Furtif** [comment]
this is correct @gegy1000  merge and test

**Comment by Furtif** [comment]
need may be revise devices nia requires ios 11.x now i thinks

File: .gitignore [hunk 3]
```diff
@@ -134,9 +134,6 @@ local.properties
 
 ## Plugin-specific files:
 
-# IntelliJ
-/out/
```
Comment (Gegy) [inline .gitignore]: Please re-add this.

File: .travis.yml [hunk 4]
```diff
@@ -7,11 +7,11 @@ branches:
   only:
   - master
   - Development
-
```
Comment (Gegy) [inline .travis.yml]: Revert these too.

File: library/.gitignore (hunk 1 above)
Comment (Gegy) [reply library/.gitignore]: Remove this, and manually exclude anything you don't want.

File: library/build.gradle [hunk 5]
```diff
@@ -4,7 +4,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.7'
-        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.1"
+        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.1"
```
Comment (Gegy) [inline library/build.gradle]: Trailing tab

File: library/build.gradle [hunk 6]
```diff
@@ -96,7 +96,7 @@ dependencies {
     compile 'com.squareup.moshi:moshi:1.2.0'
     compile 'com.annimon:stream:1.1.1'
     compile 'com.squareup.okhttp3:okhttp:3.4.1'
-    compile 'com.google.protobuf:protobuf-java:3.0.0-beta-3'
+    compile 'com.google.protobuf:protobuf-java:3.5.1'
```
Comment (Gegy) [inline library/build.gradle:99]: If these changes are happening, update the protobuf plugin too.

**Comment by Furtif** [comment]
travis passed!!! protobufs 3.5.1 is ok

File: library/build.gradle (hunk 6 above)
Comment (Furtif) [reply library/build.gradle:99]: travis passed!!! protobufs 3.5.1 is ok

**Comment by Furtif** [comment]
@gegy1000 review https://github.com/Grover-c13/PokeGOAPI-Java/pull/945#pullrequestreview-82560450

File: library/build.gradle (hunk 6 above)
Comment (Furtif) [reply library/build.gradle:99]: https://github.com/Furtif/POGOProtos/blob/master/.travis.yml#L13

**Comment by AbandonedCart** [comment]
This includes 32 commits. Did it aspire to be a branch when it grew up?

(Read: Is there going to be a merge at some point, or can this be made into a branch?)

**Comment by Grover-c13** [comment]
I'm going to merge it since there hasnt been any functional PR merged since november last year.

**Pull request merged**